akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -604,6 +612,40 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
     aeron = Aeron.connect(ctx)
   }
 
+  private def blockUntilChannelActive(): Unit = {
+    val counterIdForInboundChannel = findCounterId(s"rcv-channel: $inboundChannel")
+    retry(settings.Bind.Retries)
+
+    @tailrec def retry(retries: Int): Unit = {
+      val status = aeron.countersReader().getCounterValue(counterIdForInboundChannel)
+      if (status == ChannelEndpointStatus.ACTIVE) {
+        log.debug("Inbound channel is now active")
+      } else if (status == ChannelEndpointStatus.ERRORED) {
+        throw new RuntimeException("Inbound Aeron channel is in errored state. See Aeron logs for details.")
```

Comment:
shall we run the `AeronErrorLog` task once here, since the initial poll might not have happened yet?

Reply:
Good idea. I just hard coded it to run from time 0